features/provider/admin/backend_apis/edit.feature

```diff
@@ -0,0 +1,29 @@
+@javascript
+Feature: Backend API edit page
+
+  Background:
+    Given a provider is logged in
+
+  Scenario: Edit an existing backend API
+    Given an admin is at a backend api edit page
+    Then they will be able to edit the name, description and base url
+    But will not be able to edit its system name
```

Comment:
The scenario looks really nice. I have a concern about step conflict and discoverability. Because step `they will be able to edit the name, description and base url` is probably very backend API specific but does not indicate that it is about backend APIs. So maybe one will want to create same step for somehting else.

I don't know, does it make sense? How about 
```suggestion
    Then they will be able to edit backend api name, description and base url
    But will not be able to edit backend api system name
```

If you think that discoverability and step collisions will not be a problem, then leaving them as they are is ok.

Reply:
These particular steps were intended like this. It should work in any form, as long as it uses those fields exactly.